frontend/public/components/_dropdown.scss

```diff
@@ -91,29 +91,40 @@ $color-bookmarker: #DDD;
   display: flex;
   font-size: ($font-size-base + 1);
   height: $co-namespace-selector-mobile;
+  padding-right: 15px;
   white-space: nowrap;
-  padding: 0 15px;
 
   @media (min-width: $grid-float-breakpoint) {
     border-left: 1px solid $color-os-nav-item-seperator;
     font-size: ($font-size-base + 3);
     height: $co-namespace-selector-desktop;
-    padding-bottom: 5px;
-    padding-top: 5px;
   }
 
   &__dropdown {
-    flex: 0 1 auto;
-    max-width: 70%;
-    padding-left: 5px;
+    display: flex;
+    max-width: 95%;
   }
 
   .dropdown__not-btn {
-    align-items: baseline;
+    align-items: center;
     display: flex;
+    height: $co-namespace-selector-mobile;
+    padding-left: 15px;
+
+    @media (min-width: $grid-float-breakpoint) {
+      height: $co-namespace-selector-desktop;
+    }
+
+    &:before {
+      color: $color-pf-black-500;
```

Comment:
Can't decide if I like the color :/

@openshift/team-ux-review ?

Reply:
To me it helps set apart the label from the name.

Reply:
I feel the color contrast between the text and the bg-color is a bit too low, so did a quick WCAG check.
Turned out the original color meet the guideline but not the new one..
**original color: pf-black-300 (#d1d1d1)**
![color_ori_ d1d1d1](https://user-images.githubusercontent.com/5903705/42473514-b13747de-8379-11e8-9cc4-090aab6659b0.png)

**new color: pf-black-500 (#8b8d8f)**
![color_new_ 8b8d8f](https://user-images.githubusercontent.com/5903705/42473504-aae1aeba-8379-11e8-8508-4d32940912fb.png)

Since pf-black-500 is too dark and pf-black-400 is almost no difference here, should we just keep the same color (pf-black-300) here?   Thoughts? @cshinn , @beanh66

Reply:
Other than the color change, I think make the entire namespace selector clickable is awesome! Long waiting change!

Reply:
> Can't decide if I like the color :/

My initial impression was the same.  My vote is to keep it the same given @tlwu2013's concern regarding contrast.

Reply:
Contrast is a great Q.  I think the subtle difference is helpful, but not essential.  If it was something essential, I think it would be critical to pass an accessibility test. Since its a subtle difference to slightly improve readability, not sure it needs to pass the same test.

Curious if there are accessibility guidelines for things that are "subtle but not essential".  I'm guessing its probably not quite that sophisticated yet.

Reply:
Btw, I may have misunderstood.

- If it is `contrast is not high enough against background color`, this is a big deal.
- If it is `contrast is not significant enough between the two text colors`, this is probably not a big deal.

Thats what I meant in my previous comment.